subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,22 +34,28 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 <a name="cache"></a>
 ## Gradle's Dependency Cache can be relocated
 
-The Gradle Dependency cache under `$GRADLE_HOME/caches/modules-2` can now be relocated to another directory or host for dependencies cached by Gradle 6.1 and later. When moved to a new location or seeded into a host image, builds using the dependency cache will not need to access the network to download artifacts or metadata if the dependencies have already been downloaded. 
+The strategy of running builds in single use environments in a CI/CD setup is a challenge for build tools as a lot of state that could be reused between builds has to be rebuilt for each execution.
+Gradle has [an advanced dependency cache](userguide/dependency_resolution.html#sec:dependency_cache) which helps reduce downloads of dependencies to a minimum, inside and across build invocations.
+
+With Gradle 6.1, the dependency cache can now be relocated to another directory or host, without the need to preserve absolute paths, for dependencies cached by Gradle 6.1 and later.
+When moved to a new location or seeded into a host image, builds will be able to leverage the dependency cache and will not need to access the network to download artifacts or metadata if they are already cached. 
 
 Note that the cache should be primed and consumed by same Gradle version for maximum effect.
 See [the documentation](userguide/dependency_resolution.html#sub:cache_copy) for more details. 
 
-This is just one step along the way to help organizations using ephemeral CI agents to reduce the overhead of downloading dependencies during the build.
-
 <a name="compilation-order"></a>
-## Defining compilation order between Groovy, Scala and Java
+## Defining compilation order between JVM languages
 
-Previously, the relationship between Java, Groovy and Scala compilation was hardcoded with explicit task dependencies in the same project.
-Gradle assumed Groovy and Scala compilation would always depend on Java compilation. That is, `compileGroovy` and `compileScala` would directly depend on the output from `compileJava`.
+Combining multiple JVM languages in a single project used to have limitations around which language sources could depend on which other language.
```

Comment:
I think it could like this:
- In Gradle, you can use multiple JVM languages used in a single project (e.g. Java, Groovy, Scala, Kotlin)
- This is done by putting files in different languages in separate source sets
- Prior to Gradle 6.1 it was complicated to define the dependencies between different language sources, for example, if you wanted to make Kotlin code depend on Groovy code
- ... explain how you can do this now and provide a sample build script

Reply:
I'll rework the narrative.
For the last point, I think having a full build script sample in release notes is too much. And what's there is the relevant information.